hie-bios.cabal

```diff
@@ -183,6 +194,8 @@ test-suite bios-tests
       extra,
       tasty,
       tasty-hunit,
+      tasty-expected-failure,
+      hspec-expectations,
```

Comment:
error message isnt as nice yet as I hoped. We probably need to adapt the runner a bit.
Currently, the error message in a test-case failure looks like this (pretty formatted):
```
HUnitFailure
  (Just
     SrcLoc
       { srcLocPackage = "main"
       , srcLocModule = "Main"
       , srcLocFile = "tests/BiosTests.hs"
       , srcLocStartLine = 150
       , srcLocStartCol = 37
       , srcLocEndLine = 150
       , srcLocEndCol = 84
       })
  (Reason "Cradle loaded successfully")
```